src/components/home/components/volt_sale_card.jsx

```diff
@@ -120,12 +123,13 @@ const VoltSaleCard = () => {
         style={modalStyle}
         contentLabel='Voltage Info'
       >
+        <h1 style={{textAlign: 'center', fontSize: '25px', paddingBottom: '10px'}}>Some Text</h1>
```

Comment:
@RudeWolfCoding What is `Some Text`?

Reply:
Placeholder, got updated text from Tomas. Opening a PR soon to update it.